pkg/volume/vspherevolume/vsphere_volume_util.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/legacy-cloud-providers/vsphere/vclib"
 )
 
+// List of constants to store fixed values
 const (
 	maxRetries         = 10
```

Comment:
This constant is never used so please drop it.

Reply:
Looks like StoragePolicyName is used in checking if it is an empty string or "gold".

Reply:
I actually referred to `maxRetries`.